libraries/AP_HAL_Linux/RCOutput_Sysfs.cpp

```diff
@@ -47,11 +47,6 @@ void RCOutput_Sysfs::init()
             AP_HAL::panic("RCOutput_Sysfs_PWM: Unable to setup PWM pin.");
         }
         _pwm_channels[i]->enable(false);
-
-        /* Set the initial frequency */
-        _pwm_channels[i]->set_freq(50);
-        _pwm_channels[i]->set_duty_cycle(0);
```

Comment:
Okay, I tested and the problematic function seems to be: _pwm_channels[i]->set_duty_cycle(0);
It is enough to have only this function built in to generate a short signal for the ESCs which may end up in a short spin-up.